tests/integration/db/test_sql_data_splitting.py

```diff
@@ -49,9 +49,18 @@ def _get_connection_string_and_dialect() -> Tuple[str, str]:
 
 
 def _load_data(
-    connection_string: str, table_name: str = TAXI_DATA_TABLE_NAME
+    connection_string: str, dialect: str, table_name: str = TAXI_DATA_TABLE_NAME
 ) -> LoadedTable:
 
+    dialects_supporting_multiple_values_in_single_insert_clause: List[str] = [
+        "redshift"
+    ]
+    to_sql_method: str = (
+        "multi"
+        if dialect in dialects_supporting_multiple_values_in_single_insert_clause
```

Comment:
❤️

Reply:
I thought this was a really elegant solution to the issue. Thank you @anthonyburdi